ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ImmutableDB/StateMachine.hs

```diff
@@ -706,6 +701,11 @@ shrinkCmd Model {..} (At cmd) = fmap At $ case cmd of
     -- For simplicity, we only shrink to TipEBBs if the tip is an TipEBB,
     -- similarly for TipBlock. Otherwise we have to check whether a TipEBB is
     -- before or after a TipBlock.
+    --
+    -- TODO: Re-enable shrinker
```

Comment:
Delete commented out code and mention that we can implement it by looking at `tips`.